assets/mentions.md

```diff
@@ -38,4 +38,4 @@
 [2021/04/13](https://serokell.io/blog/rust-in-production-meilisearch) | "Rust in Production: MeiliSearch"
 [2021/04/19](https://blog.simonireilly.com/posts/serverless-search) | "Serverless search with Meilisearch and Google Cloud Run"
 [2021/05/03](http://www.skrejci.com/2021/05/meilisearch-under-windows/) | "MeiliSearch under Windows"
-[2021/05/07](https://blog.simonireilly.com/posts/serverless-search) | "Serverless search with Meilisearch and Google Cloud Run"
+[2021/05/22](https://hashnode.com/post/how-to-add-a-blazing-fast-full-text-search-capability-to-cloud-firestore-using-meilisearch-introduction-15-ckoz64twc07ppdss1hk3mfeh6) | "How to add a blazing-fast full-text search capability to Cloud Firestore using MeiliSearch"
```

Comment:
Why did we remove `blog.simonreilly...` ?

Reply:
It was already there check line 39

Reply:
Thanks!